packages/palette-tokens/src/themes/v3Dark.tsx

```diff
@@ -3,6 +3,8 @@ import { Colors, Effects, THEME as THEME_LIGHT } from "./v3";
 const COLORS: Colors = {
   /** Suitable for text on black10 and lighter */
   black100: "#ffffff",
+  /** Suitable as a background color on dark mode */
+  black90: "#121212",
```

Comment:
It should still be inverted.

Reply:
Or renamed. This doesn't make sense WRT to the inverse scale

Reply:
Like it sounds to me like you want be white100

Reply:
the new value doesn't exist in the design system. It will be mainly used for backgrounds, but it doesn't replace `white100`.

Reply:
Still doesn't make sense here.

Reply:
let me check again with Jimi maybe we can replace it everywhere.